platform/marvell-armhf/sonic-platform-et6448m/debian/sonic-platform-et6448m/DEBIAN/md5sums

```diff
@@ -0,0 +1,2 @@
+d4fe1b11c5ab1ca5f69e61d99958c0b2  usr/sbin/et6448m_plt_setup.sh
```

Comment:
generated file

Reply:
Sorry, It was my mistake. will remove all generated files.

Reply:
removed